CONTRIBUTING.md

```diff
@@ -70,32 +70,38 @@ Before submitting an issue, please:
 - Update existing tests when modifying functionality
 
 
-## Development Setup
+## Development Process
 
-- Install Go (see [go.mod](https://github.com/yaml/go-yaml/blob/main/go.mod) for
-  minimum required version)
+- Installing Go is not necessary. See "The Makefile" below.
 - Fork and clone the repository
 - Make your changes
-- Run tests and linters
-  - `make test` to run tests locally
-  - `golangci-lint run` to lint locally (requires [golangci-lint](https://golangci-lint.run/) to be installed.)
+- Run tests, linters and formatters
+  - `make test-all`
+  - `make lint`
+  - `make fmt`
+  - `make tidy`
+  - `make vet`
```

Comment:
`make vet` is not needed, `golangci-lint` also runs `govet` and will catch all errors

Reply:
will be removed in a separate PR: https://github.com/yaml/go-yaml/pull/158